lib/devise_security_extension/hooks/session_lockable.rb

```diff
@@ -4,11 +4,25 @@ def self.extract_email(env)
   rescue
     nil
   end
+
+  def self.extract_default_scope(env)
+    env['warden'].config.default_scope.safe_constantize
```

Comment:
Yes, by default User, but Hootsuite (which is a rails engine in napa) defines another scope "hs_user" which isn't an actual "class" / "model" and that cannot be constantized, that is when we fallback to "User".